### PR TITLE
Completed TF version of losses

### PR DESCRIPTION
**Pull request by jyyy03**
Converted loss from Pytorch to Tensorflow. All simple tests passed.
see loss tests in utils/loss_test.ipynb

### CODE REVIEW

**Review by wanghungjen — APPROVED** [review]
Great!! Thank you!!
